PROTO_tests/tests/client_dtypes_test.py

```diff
@@ -52,8 +52,14 @@ def test_bit_vector_error_handling(self):
         arr = ak.array([1.1, 8.3])
         with pytest.raises(TypeError):
             ak.BitVector(arr)
+    @pytest.mark.parametrize("int_types", INT_TYPES)
+    def test_bit_vector_edge_case(self, int_types):
+        arr = ak.arange(2**64-10, 2**64-1, dtype=int_types)
+        bv = ak.BitVector(arr, width=3)
```

Comment:
we need the `width` to be much larger here to actually capture the full `bitVector` for numbers this large

also doesn't look like we're actually comparing these values to anything, we're just checking that they are `BitVector`s

Reply:
Updated